sklearn/linear_model/tests/test_sag.py

```diff
@@ -465,7 +466,13 @@ def test_sag_regressor_computed_correctly(csr_container):
         max_iter=max_iter,
         random_state=rng,
     )
-    clf2 = clone(clf1)
+    clf2 = Ridge(
```

Comment:
To make the test easier to follow, I would rename `clf1` to `ridge_dense` and `clf2` to `ridge_sparse`.